_config.yml

```
@@ -178,6 +178,6 @@ fetch-remote:
     default_branch: "master"
     ref: "master"
     paths:
-      - dest: "engine/reference/builder.md"
+      - dest: "reference/dockerfile"
```

Did these need the trailing slash? (to prevent extra redirects?)

```suggestion
      - dest: "reference/dockerfile/"
```

---

Yes indeed, I just reopened this PR to remember it exists tbh :sweat_smile: but would love to have it merged. I think we still need to figure out how and when we want to move `/engine/reference` to `/reference` first.